src/firefly/java/edu/caltech/ipac/firefly/data/HttpResultInfo.java

```diff
@@ -0,0 +1,50 @@
+package edu.caltech.ipac.firefly.data;
+/**
+ * User: roby
+ * Date: 12/21/20
+ * Time: 10:10 AM
+ */
+
+
+import edu.caltech.ipac.util.StringUtils;
+import edu.caltech.ipac.util.download.ResponseMessage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Trey Roby
+ */
+public class HttpResultInfo {
```

Comment:
HttpResultInfo and FileInfo is related but has not object-oriented relationship is a bit awkward.  But, I can't think of a simple alternative.

Reply:
I thought about it, an OO relationship would have been force and did not make sense.